tests/specs/check/typecheck_doc_duplicate_identifiers/mod.out

```diff
@@ -0,0 +1,2 @@
+Check [WILDCARD]/tests/specs/check/typecheck_doc_duplicate_identifiers/mod.ts
```

Comment:
It might be better to just do `[WILDCARD]/mod.ts` so that these aren't coupled so much with the spec test structure (and below for all the tests). That will make the test more flexible in the future (ex. it allows easily doing `"tempDir": true` or moving the tests around to a new folder without them failing).

Reply:
That makes lots of sense. Fixed.